### PR TITLE
set path in MATLAB wrapper so that you can call it from any folder

### PR DESCRIPTION
**Pull request by sg-s**
I updated the wrapper so that it adds to path the folder that contains the binary. this way, you can run `fast_tsne` from any path, not just the one that contains the binary.

### CODE REVIEW

File: fast_tsne.m [hunk 1]
```diff
@@ -47,6 +47,17 @@
 % IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY 
 % OF SUCH DAMAGE.
 
+    % rotate the input so that it makes sense
+    if size(X,1) < size(X,2)
+        X = X';
+    end
```
Comment (lvdmaaten) [inline fast_tsne.m]: I don't think this is a good idea. Some people may have situations in which D > N, and then run into weird problems.

**Comment by lvdmaaten** [comment]
Looks nice! Can you remove the transpose as it may lead to bugs for other users (e.g., in NLP you are commonly in the scenario where D > N).

File: fast_tsne.m (hunk 1 above)
Comment (sg-s) [inline fast_tsne.m]: OK, will do.

**Comment by sg-s** [comment]
done!

**Pull request merged**